src/guides/v2.4/rest/tutorials/inventory/create-shipment.md

```diff
@@ -13,21 +13,21 @@ functional_areas:
   - Integration
 ---
 
-In the previous step, the SSA recommended shipping 20 `24-WB01` items and 19 `24-WB03` items from Baltimore. It also recommended shipping 31 `24-WB03` items from Reno.
+In the previous step, the SSA recommended shipping 35 `24-WB01` items and 20 `24-WB03` items from the Northeast warehouse. It also recommended shipping 5 `24-WB01` items from the Brooklyn store.
 
-You can always override the SSA recommendations. It would be valid, for example, to ship all the `24-WB01` items and 42 `24-WB03` items from Reno, leaving 8 `24-WB03` items to be shipped from Baltimore.
+You can always override the SSA recommendations. It would be valid, for example, to ship 25 `24-WB01` items and 5 `24-WB03` items each from the Brooklyn, Manhattan, and Huntingdon stores.
```

Comment:
```suggestion
You can always override the SSA recommendations. It would be valid, for example, to ship 25 `24-WB01` items and 5 `24-WB03` items each from the Brooklyn, Manhattan, and Huntington stores.
```